lib/oban/postgres_notifier.ex

```diff
@@ -0,0 +1,176 @@
+defmodule Oban.PostgresNotifier do
+  @doc """
+  Postgres Listen/Notify based Notifier
```

Comment:
This should contain the old caveats section. It's still the default notifier most apps will use.

Reply:
True! Removed from one place and forgot to add in the other